app/browser/tools/timestampCopyOverride.js

```diff
@@ -0,0 +1,84 @@
+const ReactHandler = require("./reactHandler");
+
+class TimestampCopyOverride {
+    constructor() {
+        this.overrideInterval = null;
+    }
+
+    init(config) {
+        this.config = config;
+        this.overrideInterval = setInterval(() => this.applyOverride(), 1000);
+    }
+
+    stop() {
+        if (this.overrideInterval) {
+            clearInterval(this.overrideInterval);
+            this.overrideInterval = null;
+        }
+    }
+
+    applyOverride() {
+        const coreServices = ReactHandler._getTeams2CoreServices();
+        if (!coreServices || !coreServices.coreSettings) return;
```

Comment:
as sonarqube says:

"Prefer using an optional chain expression instead, as it's more concise and easier to read."

```suggestion
        if (!coreServices?.coreSettings) return;
```
```